draft-fossati-tls-attestation.md

```diff
@@ -154,74 +154,108 @@ Several formats for encoding evidence are available, such as:
-to verify the CertificateVerify message.
-
-# Use of Evidence with the Background Check Model
-
-The background check model is described in Section 5.2 of 
-{{I-D.ietf-rats-architecture}} and allows the following modes
-of operation when used with TLS, namely:
+As typical with new features in TLS, the client indicates support for the new
+extension in the ClientHello message. The newly introduced extensions allow
+remote attestation credentials and nonces to be exchanged. The nonces are used
+for guaranteeing freshness of the exchanged evidence when the background check
+model is in use.
+
+When either the evidence or the results extension is successfully negotiated,
+the content of the corresponding Certificate message contains a payload that is
+encoded based on the wrapper defined in {{I-D.ftbs-rats-msg-wrap}}.
+
+In TLS a client has to demonstrate possession of the private key via the
+CertificateVerify message, when client-based authentication is requested. The
+attestation payload must contain assertions relating to the client's TLS
+Identity Key (TIK), which associate the private key with the attestation
+information. These assertions may come in the form of a Key Attestation Token
+(KAT), or of specific claims in an attestation result document. An example of a
+KAT format utilizing the EAT format can be found in {{I-D.bft-rats-kat}}.
+
+The relying party can obtain and appraise the remote attestation results either
+directly from the Certificate message (in the passport model), or by relaying
+the evidence from the Certificate message to the verifier. Subsequently, the
+attested key is used to verify the CertificateVerify message.
+
+When using the passport model, the remote attestation results obtained by the
+attester from its trusted verifiers can be cached and used for any number of
+subsequent TLS handshakes. This allows the two sides to amortize the cost of
+remote attestation by reducing the frequency with which evidence must be
+produced and appraised. However, this does create a risk of stale attestation
+results being used for authentication. The relying party must therefore be
+empowered to reject an authentication attempt with results that do not meet its
+staleness policy. This should trigger a new evidence appraisal on the attester
+side.
+
+# Use of Remote Attestation credentials in the TLS handshake
+
+For both the passport model (described in section 5.1 of {{RFC9334}}) and
+background check model (described in Section 5.2 of {{RFC9334}}) the following
+modes of operation are allowed when used with TLS, namely:
 
 - TLS client is the attester, 
 - TLS server is the attester, and
 - TLS client and server mutually attest towards each other. 
 
-We will show the message exchanges of the three cases in
-sub-sections below.
+We will show the message exchanges of the first two cases in sub-sections below.
+Mutual authentication via attestation combines these two (non-interfering)
```

Comment:
Why not simply "Mutual attestation"?

Reply:
@muhammad-usama-sardar Because from a TLS point of view this is authentication.